doc/fgpu.design.rst

```diff
@@ -566,6 +558,26 @@ actually computed.
    real signal into a complex signal, the subsampling factor is twice the
    decimation factor.
 
+The first three steps are implemented by a new "digital down-conversion"
```

Comment:
It's not recycled from wideband.

Reply:
Bike-shedding, but I don't think "new" adds much value here because there isn't a DDC kernel in the wideband case.

I'd personally rephrase as e.g. "not shared with the wideband" or just leave it out. But I don't feel strongly enough about it that I would insist on it being changed, if you think it's better as it currently stands.

Reply:
I've removed it.